### DISCUSSION TITLE
GLaDOS Potato

### DISCUSSION THREAD

**Discussion by CN1o2** [BEEmod/BEE2-items#3861] [Feature Suggestions]
Can you (if possible) add potato GLaDOS and your voice lines?

Thus complementing the maps with Wheatley and also those of the old Aperture. For example: put it on the ground for the player to pick up and place on their portal device. I think it's a good idea, it would complement the maps (Of course, if that's possible).

**Comment by vrad-exe** [comment]
See #3339. PotatOS would just appear when specific voice packs were selected, otherwise you could end up with nonsensical situations where normal GLaDOS is talking but she's also on your portal gun.
